website/docs/d/iam_trusted_profiles_link.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_link"
+description: |-
+  Get information about iam_trusted_profile_link
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_link
+
+Provides a read-only data source for iam_trusted_profile_link. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile link, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link
```

Comment:
Provides a read-only data source for iam_trusted_profile_link. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile link, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link

as

Retrieve information about IAM trusted profile link as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information, about trusted profile link, see [Create a link to a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link)